Trackovid19-web/src/app/app.component.ts

```diff
@@ -25,4 +30,15 @@ export class AppComponent {
 
     this.renderer2.appendChild(this._document.body, s);
   }
+
+  ngOnInit() {
+    //TODO: Check if we want to go with this or not
+    if (this.swUpdate.isEnabled) {
+      this.swUpdate.available.subscribe(() => {
+        if (confirm('New version available. Load New Version?')) {
+          window.location.reload();
```

Comment:
To update the PWA to a new version, a simple `window.location.reload()` won't do it.

Reply:
From what I've seen until know it works. But probably I'm forgetting something.
Can you tell me what is need?

Reply:
The gist of it is, first we send a message to the current service worker telling to activate the new service worker and them we reload the page.

Reply:
I'm not following you know. 
The service work is the same, what changes is the `ngsw.json` which have the list of all files and versions. Every time we enter the page, we check the file, and if something change `this.swUpdate.available` is triggered and we just need to reload the page, since the new assets were download in the meanwhile.

Reply:
Technically there are two different service worker instances, the currently being used and the new one waiting to take place of the current one.

We only need a simple page refresh if the service worker `skipWaiting` method was already called before (this method replaces the currently running service worker with the new one), which I'm not sure with the Angular service worker configuration already does that for us on the `install` service worker event.

Reply:
Ok, now I'm understanding your concern and I'm following you. 
The angular service worker is doing that for us 👍 
```
...
this.scope.addEventListener('install', (event) => {
                // SW code updates are separate from application updates, so code updates are
                // almost as straightforward as restarting the SW. Because of this, it's always
                // safe to skip waiting until application tabs are closed, and activate the new
                // SW version immediately.
                event.waitUntil(this.scope.skipWaiting());
            });
... 
```

Reply:
Nice, looks like everything is ready to go 🚀.

Reply:
Nice. Thanks for your review!